nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -76,7 +77,11 @@ def test_plot_functions():
     for func in [plot_anat, plot_img, plot_stat_map,
                  plot_epi, plot_glass_brain,
                  partial(plot_stat_map, symmetric_cbar=True),
-                 partial(plot_stat_map, symmetric_cbar=False)]:
+                 partial(plot_stat_map, symmetric_cbar=False),
+                 partial(plot_stat_map, symmetric_cbar=False, vmax=10),
+                 partial(plot_stat_map, symmetric_cbar=True, vmax=10),
+                 partial(plot_stat_map, colorbar=False)
```

Comment:
Should we try to add similar tests for plot_stat_map where the data has both positive and negative values?